sycl/test/check_device_code/native_cpu/native_cpu_attrs.cpp

```diff
@@ -0,0 +1,30 @@
+// REQUIRES: native_cpu_ock
+// RUN: %clangxx -fsycl -fsycl-targets=native_cpu -Xclang -sycl-std=2020 -mllvm -sycl-native-dump-device-ir %s | FileCheck %s
+
+#include "sycl.hpp"
+using namespace sycl;
+
+class Test;
+int main() {
+  sycl::queue deviceQueue;
+  sycl::nd_range<2> r({1, 1}, {
+                                   1,
+                                   1,
+                               });
```

Comment:
This is some odd formatting. I think the formatter is broken, so could you please make sure the formatting is in line with clang-format?